src/components/profile/ProjectItem.tsx

```
@@ -42,85 +47,96 @@ export default function ProjectItem({
-          />
-        </div>
+    <div className="mb-8">
+      <div className="flex gap-6 relative">
+        {/* 상품 이미지 */}
+        <div className="relative w-60 h-60 flex-shrink-0">
+          {/* 상품 이미지 클릭 시 상품 상세 페이지로 이동 */}
+          <div className="w-full h-full cursor-pointer" onClick={handleProjectClick}>
+            <Image
+              src={imageUrl || "/placeholder.svg"}
+              alt={projectName}
+              width={256}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)